dolphin/api/v1/pools.py

```diff
@@ -12,19 +12,79 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+import six
+from webob import exc
+
+from dolphin import db, exception
+from dolphin.api import common_utils as common
 from dolphin.api.common import wsgi
+from dolphin.api.views import pools as pool_view
+from dolphin.i18n import _
 
 
 class PoolController(wsgi.Controller):
+    def _verify_storage(self, context, storage_id):
```

Comment:
better to have this as bool function as it checks existence of storage to move further